openedx/core/djangoapps/api_admin/models.py

```diff
@@ -35,6 +42,8 @@ class ApiAccessRequest(TimeStampedModel):
     reason = models.TextField(help_text=_('The reason this user wants to access the API.'))
     company_name = models.CharField(max_length=255, default='')
     company_address = models.CharField(max_length=255, default='')
+    base_url = models.CharField(max_length=255, default='')
```

Comment:
`get_absolute_url` is only available on the request object so in order to create proper links in the emails we need to store the base_url.  Another approach is to use a method in the django sites package.  Not sure if I have a preference.  I think I would prefer django sites, but we aren't using sites that way in edx-platform at all so I don't know what the impact would be.  This was the more surgical approach